projects/dex-ui/src/components/Liquidity/SlippagePanel.tsx

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+import React from "react";
+import styled from "styled-components";
+
+import arrowLeft from "/src/assets/images/arrow.svg";
+
+type SlippagePanelProps = {
+  slippageValue: number;
+  slippageValueChanged: (value: string) => void;
+  closeButtonClicked: () => void;
+};
+
+const SlippagePanel = ({ slippageValueChanged, closeButtonClicked, slippageValue }: SlippagePanelProps) => {
+  const buttonClickHandler = () => {
+    closeButtonClicked();
+  };
```

Comment:
no need for this, you can pass the prop to onclick directly:
`<img src={arrowLeft} alt="Back" onClick={closeButtonClick} />`